### ISSUE TITLE
error because relation "airport_infos" does not exist

### ISSUE THREAD

**Issue by masterAllen** [tumi8/hloc#9]
Hi, thank you very much for your open code!

I try to use it, but i get an error that relation "airport_infos" does not exist. I have collected all data and renamed it correctly.
This error happened when i executed script `example-inital-db-setup.sh`.

Here is the screenshot.

![2022-04-16-152422_1920x1080_scrot](https://user-images.githubusercontent.com/26152844/163666292-1336f9f1-ce81-492d-8a70-0d55ee60d683.png)

I think i have missed some steps about database but i dont know how can i do now. 
Thank you in advance.

**Comment by sattler** [comment]
Hi I am sorry but I cannot help without more context and description of what you did and how the error manifests.

**Comment by masterAllen** [comment]
Thank you for your reply firstly. 

I have solve this issue by creating some tables, e.g., `airport_infos`.

My steps are as follows.
1. Install and config postgres.
2. collect data such as CLLI, UNLOCODE.
3. modify `example-init-db-setup` to run it in my custom directory. 
4. run `example-init-db-setup`.

I got `relation airport_infos does not exist` error when i execute step4. I think maybe i missed some steps.

I have created some tables such as `airport_infos`, `locations` and fixed this error.
For example, here are the sql commands to create `airport_infos`:
```sql
create table airport_infos (
id serial PRIMARY KEY,
iata_codes VARCHAR(5000),
icao_codes VARCHAR(5000),
faa_codes VARCHAR(5000)
);
```
Thank you very much for this great project!
